### ISSUE TITLE
Option `IBM 7.3` not available when latest deployment `Select a rule set deployment date` is selected

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1926]
- I can only select `IBM Accessibility 7.3` when the Rule Set is set on `Latest Deployment` or `Preview rules`. 
- When set on `06 June 2024 Deployment (IBM 7.2, 7.3)`, the only option is `IBM Accessibility 7.2`.  Both v7.2 and v7.3 should be available.
- If `IBM Accessibility 7.3` was previously selected with `Latest Deployment` or `Preview rules`, but then the user changes and selects `06 June 2024 Deployment (IBM 7.2, 7.3)`, the Guideline is automatically changed (no notification either) by the Checker to `IBM Accessibility 7.2` instead.  v7.3 should remain.
- Behaves the same in FF and Chrome.

![image](https://github.com/IBMa/equal-access/assets/16193609/31ce9319-37c4-4ebf-9743-c05079fe956b)

![image](https://github.com/IBMa/equal-access/assets/16193609/7ff95f54-81f6-4015-90d6-c1cba4be1412)

**Comment by shunguoy** [comment]
@MHoov  The team thinks this is a low-priority issue because an alternative exists. If a user selects "Latest Deployment", the 7.3 will appear.

**Comment by philljenkins** [comment]
This behavior listed in the issue needs to be re-evaluated when the next Rule Set is deployed and there is a second deployment date with both v7.2 and v7.3. 
```
Latest Deployment
xx June 2024 Deployment (IBM 7.2, 7.3)
06 June 2024 Deployment (IBM 7.2, 7.3)
09 May 2024 Deployment (IBM 7.2)
...
```

**Comment by philljenkins** [comment]
Actually, this is now working correctly with the `17 June 2024` rule set.
@MHoov So this can be closed.

![Screenshot 2024-06-20 at 4 53 29 PM](https://github.com/IBMa/equal-access/assets/16193609/f2db5f36-3603-4cbd-9847-060b51d82c33)
